source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
+=================================================
+
+It's a relatively straight forward process to create an account on Mattermost and start communicating with other people. You enter some basic information on the account creation page then click **Create Account**.
+
+.. Important::
+  - The account creation process might be different than described here, depending on the policies of your organization.
+  - If your administrator sent you instructions, use those instructions.
+  - If you are not able to create an account using these instructions, contact your administrator.
+
+On the account creation page, you must enter the following information:
```

Comment:
I think we should add a section here about the options for signing up other than email. Sometimes the system could have more than one account creation method enabled, then the user should follow instructions from their System Admin. Open to suggestions to build on this idea, but I think the following info is important:

1. Email and Password
  - What you have here
2. Single-Sign-On Services
  - Mention that accounts are created the first time you sign in.
  - Username is automatically generated but can be changed in Account Settings for certain account types.
  - Options for SSO are GitLab, LDAP, SAML, Office 365 and Google depending on what the sys admin has enabled.